docs/syntax/frontmatter.md

```diff
@@ -43,43 +43,28 @@ See [](./applies.md)
-| `cloud-kubernetes`                          | Elastic Cloud Kubernetes                      |
-| `cloud-serverless`                          | Elastic Cloud Serverless                      |
-| `cloud-terraform`                           | Elastic Cloud Terraform                       |
-| `ecs`                                       | Elastic Common Schema (ECS)                   |
-| `ecs-logging`                               | ECS Logging                                   |
-| `edot-cf`                                   | EDOT Cloud Forwarder                          |
-| `edot-sdk`                                  | Elastic Distribution of OpenTelemetry SDK     |
-| `edot-collector`                            | Elastic Distribution of OpenTelemetry Collector |
-| `elastic-agent`                             | Elastic Agent                                 |
-| `elastic-serverless-forwarder`              | Elastic Serverless Forwarder                  |
-| `elastic-stack`                             | Elastic Stack                                 |
-| `elasticsearch`                             | Elasticsearch                                 |
-| `elasticsearch-client`                      | Elasticsearch Client                          |
-| `filebeat`                                  | Filebeat                                      |
-| `fleet`                                     | Fleet                                         |
-| `heartbeat`                                 | Heartbeat                                     |
-| `integrations`                              | Integrations                                  |
-| `kibana`                                    | Kibana                                        |
-| `logstash`                                  | Logstash                                      |
-| `machine-learning`                          | Machine Learning                              |
-| `metricbeat`                                | Metricbeat                                    |
-| `observability`                             | Elastic Observability                         |
-| `packetbeat`                                | Packetbeat                                    |
-| `painless`                                  | Painless                                      |
-| `search-ui`                                 | Search UI                                     |
-| `security`                                  | Elastic Security                              |
-| `winlogbeat`                                | Winlogbeat                                    |
+
+:::{include} /_snippets/products-list.md
+:::
+
+`products` can also be defined in the [`docset.yml` file](/configure/content-set/navigation.md#products).
+If you define `products` in a page's Markdown file and the `docset.yml` file also includes `products`, docs-builder will combine the two lists.
+You can _not_ override doc set level `products` at the page level.
```

Comment:
Since this seems to be repeated, shall we add it to the include?

Reply:
The general idea is repeated, but the content is not identical: one is on the frontmatter page calling out that it's also possible to define products in the `docset.yml` file and the other in on the navigation page calling out that it's also possible to define in the frontmatter.